src/mediumComplexityProblem9/FindOddNumber.java

```diff
@@ -0,0 +1,21 @@
+package mediumComplexityProblem9;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FindOddNumber {
+    public Integer getElement(List<Integer> list, Integer indexOfOddNo) {
```

Comment:
use int datatype for indexOfOddNo

Reply:
Updated datatype as 'int' for 'indexOfOddNo'